editor/editor_settings.cpp

```diff
@@ -599,8 +599,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	hints["global/default_project_export_path"] = PropertyInfo(Variant::STRING, "global/default_project_export_path", PROPERTY_HINT_GLOBAL_DIR);
 	set("interface/show_script_in_scene_tabs", false);
 
-	set("text_editor/theme/color_theme", "Default");
-	hints["text_editor/theme/color_theme"] = PropertyInfo(Variant::STRING, "text_editor/theme/color_theme", PROPERTY_HINT_ENUM, "Default");
+	set("text_editor/theme/color_theme", "Adaptive");
+	hints["text_editor/theme/color_theme"] = PropertyInfo(Variant::STRING, "text_editor/theme/color_theme", PROPERTY_HINT_ENUM, "Adaptive,Custom");
```

Comment:
Might want to keep this one as "Adaptive,Default" to be safe unless you want to change everywhere to use "Custom" (it gets overwritten anyway).

Reply:
oh true.